.github/workflows/driver_ci.yaml

```diff
@@ -0,0 +1,135 @@
+name: Driver Build
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  prepare_env:
+   name: "Prepare env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9.7'
```

Comment:
could you remind me what is this for?
it's hard to tell which setup is required by which logic

Reply:
is it for `docker/build-push-action`?